components/StatusDropdown/index.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'; //REMOVE
+import { FilterType } from '@/types/schema';
+import { ExitIcon } from '../../assets/Dropdown-Icons/icons';
+import COLORS from '../../styles/colors';
+import {
+  ApplyFiltersText,
+  FilterNameText,
+  FilterOptionsText,
+} from '../../styles/texts';
+import {
+  ApplyButtonStyles,
+  ButtonStyles,
+  ButtonWithIconStyles,
+  CheckboxContainer,
+  CheckboxStyles,
+  ExitStyles,
+  FilterDropdownStyles,
+  OptionTitleStyles,
+} from './styles';
+
+interface StatusDropdownProps {
+  selectedStatus: string[];
+  setSelectedStatus: (status: string[]) => void;
+  handleButtonClick: (filter: FilterType) => void;
+  icon: React.ReactNode;
+  label: string;
+  currFilter: FilterType;
+}
+
+export default function StatusDropdown({
+  selectedStatus,
+  setSelectedStatus,
+  handleButtonClick,
+  icon,
+  label,
+  currFilter,
+}: StatusDropdownProps) {
+  const filterOptions = [
+    { title: 'In Progress', color: `${COLORS.ashGrey}` },
+    { title: 'Operational', color: `${COLORS.chateauGreen}` },
+  ];
+
+  const handleStatusChange = (status: string) => {
+    setSelectedStatus(selectedStatus[0] === status ? [] : [status]);
+  };
+
+  const isApplyButtonActive = selectedStatus.length > 0;
+
+  return (
+    <FilterDropdownStyles>
```

Comment:
below this, add <FilterContentDiv> to wrap around all the filter content

Reply:
Should look like this:
<img width="557" alt="Screenshot 2024-11-16 at 3 37 53 PM" src="https://github.com/user-attachments/assets/edf88669-3215-4e39-94ea-643fe4dce959">